src/main/java/com/gdschongik/gdsc/domain/coupon/domain/IssuanceMethodType.java

```diff
@@ -0,0 +1,13 @@
+package com.gdschongik.gdsc.domain.coupon.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum IssuanceMethodType {
```

Comment:
```suggestion
public enum IssuanceType {
```
`Method` 와 `Type` 이 의미 상 같은 표현으로 보입니다